samples/nrf9160/http_application_fota/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "HTTP application update sample"
+
+config DOWNLOAD_HOST
+	string "Download host name"
+
+config DOWNLOAD_FILE
+	string "The file to download"
+
+config FLASH_WRITE_OFFSET
+	hex
+	default 0x80000
+
+config APPLICATION_VERSION
+	int
+	default 1
```

Comment:
This could use the `range 1 2` syntax to make sure an invalid config is not set(however I think it's handled fine by the `#ifdef != 1` in the code.